packages/varlet-ui/src/form/docs/en-US.md

```diff
@@ -254,6 +254,7 @@ bindForm?.(apis)
 | --- | --- | --- | --- | 
 | `readonly` | Whether the readonly | _boolean_ | `false` |
 | `disabled` | Whether the disabled | _boolean_ | `false` |
+| `scroll-to-first-error` | Whether to scroll to the first error field when validation failed | _boolean_ | `false` |
```

Comment:
please consider, I want the prop is `scroll-to-error` and support `start` and `end`.

Reply:
> please consider, I want the prop is `scroll-to-error` and support `start` and `end`.

Do the `start` and `end` refer to the scrollIntoView option?

maybe prop type are `start` `end` `none`?

Reply:
1. scroll to the `first` error form item when the scroll-to-error="start"
2. scroll to the `last` error form item when the scroll-to-error="end"